lib/debugging.ts

```diff
@@ -5,6 +5,41 @@ import * as path from 'path';
 import {driver} from './index';
 import {getEnabledLogTypes, saveLogs} from './logs';
 
+function getTestSuiteHierarchy(test: Mocha.Runnable): Mocha.Suite[] {
+  const suites: Mocha.Suite[] = [];
+  let parent = test.parent;
+  while (parent) {
+    suites.push(parent);
+    parent = parent.parent;
+  }
+  return suites;
+}
+
+function sanitizeStringForFilename(name: string): string {
+  return name
+      .replace(/\s/g, '-')
+      .replace(/[^A-Za-z0-9]/g, '')
+      ;
```

Comment:
```suggestion
    .replace(/\s/g, '-')
    .replace(/[^A-Za-z0-9]/g, '')
    ;
```

Indentation (and below).